src/store/books/book.js

```diff
@@ -12,16 +12,22 @@ export const addBook = ({ title, author }) => ({
   },
 });
 export const removeBook = (id) => ({
-  type: BOOK_ADDED,
+  type: BOOK_REMOVED,
```

Comment:
- [ ] I cannot understand why you used store folders instead of reducx. Please update this.

Reply:
!!!

Reply:
You have a folder name store but it should be redux

Reply:
So because the folder name is "store" and is not "redux" ?
I could argue that we should not use the name of the tools we use as artifacts in projects; naming should be based on their role